Dockerfile

```diff
@@ -13,4 +13,6 @@ WORKDIR /var/app
 
 RUN npm install
 
+RUN npm config set script-shell /bin/bash

```

Comment:
We can just make bash the entrypoint so that everything runs under bash properly.  As I mentioned offline, supervisord is also superflous.

Do you mind if I just open a PR myself for you to review and ask questions about? I barely knew anything about Docker when I wrote this and there's a lot of low-hanging fruit.

Reply:
Go right ahead!